packages/pgx/pgx.2.2/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "Pure-OCaml PostgreSQL client library"
+description:
+  "PGX is a pure-OCaml PostgreSQL client library, supporting Async, LWT, or synchronous operations."
+maintainer: ["Arena Developers <silver-snakes@arena.io>"]
+authors: ["Arena Developers <silver-snakes@arena.io>"]
+license: "LGPL-2.0-only with OCaml-LGPL-linking-exception"
```

Comment:
```suggestion
license: "LGPL-2.0-only WITH OCaml-LGPL-linking-exception"
```

And same for all other packages. This follows the standard notation of licenses.